sp-docs/docs/quickstart.rst

```diff
@@ -1,55 +1,64 @@
+.. _quickstart:
+
 Quickstart guide
 ----------------
 
-An initial set of documentation can be built directly from a clone of this
-repository.
+The following steps will guide you through setting up your documentation using the starter pack and building an initial set of documentation.
+
+For more detailed information, see the other sections of the starter pack documentation.
+
+1. Download the :literalref:`init.sh<https://raw.githubusercontent.com/canonical/sphinx-docs-starter-pack/use-canonical-sphinx-extension/init.sh>` file from the starter pack repository and place it into the repository where you want to set up your documentation.
+
+#. Run the script and specify where you want your documentation files to be placed.
+
+   See :ref:`enable` for detailed information on what the script does.
+
+#. Enter the documentation folder (the folder you specified when running the script) and build the documentation with the following command::
+
+     make run
+
+   This command creates a virtual environment, installs the Python dependencies, builds the documentation, and serves it on :literalref:`http://127.0.0.1:8000/`.
 
-First, clone this repository to a local directory, and change to this
-directory:
+   See :ref:`build` for detailed information on different commands for building and viewing the documentation.
 
-.. code-block:: sh
+#. Keep this session running to rebuild the documentation automatically whenever a file is saved, and open :literalref:`http://127.0.0.1:8000/` in a web browser to see the locally built and hosted HTML.
 
-   git clone git@github.com:canonical/sphinx-docs-starter-pack <new-repository-name>
-   cd <new-repository-name>
+#. To add a new page to the documentation, create a new document called :file:`reference.rst`, insert the following |RST|-formatted ``Reference`` heading, and then save the file:
```

Comment:
It only works on some pages and not others. I think because the definition isn't included generally (e.g. with the reusable links), it needs to be specified on every page (and it's only on some pages)

Reply:
It works fine for me. And it **is** included on all pages (since it is actually defined in the `links.txt` file).